app/views/form/show.html.erb

```diff
@@ -1,2 +1,2 @@
-<div id="respond-to-form" data-form-id="<%= @form_id %>" data-form-display="<%= @form_display %>">
+<div id="respond-to-form" data-form="<%= @form %>" data-form-id="<%= @form_id %>" data-form-display="<%= @form_display %>">
```

Comment:
if there are double quotes in the form json this will likely break right? Not sure, seems like it.